src/CMakeLists.txt

```diff
@@ -139,6 +139,6 @@ if (WIN32)
 endif()
 
 add_subdirectory(tool)
-add_subdirectory(test)
 add_subdirectory(platform)
```

Comment:
nit: if we're going to move these around, can we make them alphabetical?

Reply:
Currently they have to be ordered a certain way because of dependencies.